src/traces/bar/index.js

```diff
@@ -9,38 +9,29 @@
 
 'use strict';
 
-var Plotly = require('../../plotly');
```

Comment:
In all the `traces/*/index.js`, we can now get rid of the `Plotly` dependency, and no longer call `Plots.register` - instead calling it from the `register` function in `src/index.js`.